### PR TITLE
Landing page temp placeholder

### PR DESCRIPTION
**Pull request by nate-ni**
# Pull Request

## 🤨 Rationale
Quick Update to nimble.ni.dev while we wait on final design.

## 🧪 Testing
verified locally

## ✅ Checklist
- [ ] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: packages/site/root/styles.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+body{
```
Comment (jattasNI) [inline packages/site/root/styles.css:1]: Can you do a quick formatting pass on this file? The things that jump out at me are:
1. inconsistent spacing before `{` blocks 
2. inconsistent spacing after `:`s
3. inconsistent indentation

Longer term we should enable linting or prettier on CSS in this directory but that needn't block this submission.

File: packages/site/src/index.html [hunk 2]
```diff
@@ -7,19 +7,43 @@
...
+        <div class="cards">
+            <a class="card" href="https://github.com/ni/nimble">
+                <div class="icon"><img src="./gh.svg" alt="Nimble Logo"></div>
+                <div class="name">GitHub</div>
+                <div class="description">Open source access to code snippets</div>
+            </a>
+            <a class="card" href="https://nimble.ni.dev/storybook/">
+                <div class="icon"><img src="./sb.svg" alt="Nimble Logo"></div>
+                <div class="name">Storybook</div>
+                <div class="description">Search and retrieve components</div>
```
Comment (jattasNI) [inline packages/site/src/index.html:34]: "retrieving" components doesn't really resonate with me. Here's an alternate suggestion but feel free to leave as-is or change to something else.
```suggestion
                <div class="description">Live playground for component APIs</div>
```

**Review by jattasNI — APPROVED** [review]
I'm ok if you merge this without @rajsite's approval since it's "launching" tomorrow morning. We can address his feedback in a follow up PR if there is any.

File: packages/site/root/styles.css (hunk 1 above)
Comment (nate-ni) [reply packages/site/root/styles.css]: cleaned up in https://github.com/ni/nimble/pull/800/commits/0ecc3883eeebfcc1ca6b9615d2fabca347f21a66

**Pull request merged**